template/client/store/index.js

```diff
@@ -1,77 +1,16 @@
-// @flow
```

Comment:
Disabling `flow` is not required anymore, when https://github.com/wemake-services/wemake-vue-template/issues/492 is solved.

Can you please enable it once again?